raster/src/main/scala/geotrellis/raster/io/geotiff/writer/TiffTagFieldValue.scala

```diff
@@ -6,8 +6,10 @@ import geotrellis.raster.io.geotiff.tags.codes._
 import geotrellis.raster.io.geotiff.tags.codes.TagCodes._
 import geotrellis.raster.io.geotiff.tags.codes.TiffFieldType._
 import geotrellis.vector.Extent
-
 import java.nio.ByteOrder
+
+import geotrellis.raster.render.IndexedColorMap
```

Comment:
Got it. Again, out of habit hit my "organize formats" keybinding, which I have set up differently.